management.yml

```diff
@@ -216,6 +216,8 @@ paths:
     get:
       security:
       - ztSession: []
+      - oauth2:
+        - openid
```

Comment:
The OIDC was missing on these endpoints, causing OIDC bearer tokens to not work for auth.